src/workspace-data/data-table/uri-viewer/UriDownloadButton.ts

```diff
@@ -76,7 +76,7 @@ export const UriDownloadButton = ({ uri, metadata: { bucket, name, fileName, siz
       ButtonPrimary,
       {
         disabled: !url,
-        href: url,
+        href: url || undefined,
```

Comment:
`url` is `string | null`, but Clickable wants `string | undefined`.

Reply:
ditto on an inline explanation